website/pages/docs/secrets/databases/index.mdx

```diff
@@ -124,25 +126,27 @@ the proper permission, it can generate credentials.
     lease_id           database/creds/my-role/2f6a614c-4aa2-7b19-24b9-ad944a8d4de6
     lease_duration     1h
     lease_renewable    true
-    password           8cab931c-d62e-a73d-60d3-5ee85139cd66
-    username           v-vaultuser-e2978cd0-
+    password           FSREZ1S0kFsZtLat-y94
+    username           v-vaultuser-e2978cd0-ugp7iqI2hdlff5hfjylJ-1602537260
     ```
 
 ## Database Capabilities
+As of Vault 1.6, all databases support dynamic roles, static roles, and rotating the root user's credentials.
+
 | Database | Root Credential Rotation | Dynamic Roles | Static Roles |
 | --- | --- | --- | --- |
```

Comment:
Might also want to format/extend these two lines like you're doing with the other tables.